docker-compose.gcb.yml

```diff
@@ -17,7 +17,7 @@ x-test-common: &test-common
     - ".artifacts:/.artifacts"
   environment:
     SNUBA_SETTINGS: "$SNUBA_SETTINGS"
-    CLICKHOUSE_HOST: clickhouse
+    CLICKHOUSE_HOST: clickhouse.dev.ci
```

Comment:
`.ci` is a *real* TLD (I know because I own a few of them!) -- please pick something that isn't real or use the example domain (`.local` I believe is the standard answer for this)